crates/physical-plan/src/plan.rs

```diff
@@ -69,11 +161,15 @@ pub enum IndexOp {
 /// Then the complexity of the index join is `n * f(m)`
 pub struct IxJoin {
     /// The lhs input used to probe the index
-    pub input: Box<PhysicalPlan>,
+    pub lhs: Box<PhysicalPlan>,
     /// The rhs indexed table
-    pub table: Arc<TableSchema>,
-    /// The rhs index
-    pub index: IndexId,
+    pub rhs: Arc<TableSchema>,
+    /// The rhs field name
+    pub rhs_label: Symbol,
+    /// The index id
```

Comment:
I think it will be nice if all the index info is in their own `struct`:

```rust
struct Idx {
index_id: IndexId,
index_cols: ColList,
unique: bool,
}
```

... it will simplify later when we need to know for example if the `idx` comes from a `btree` or a `hash`.